MetaMask/metamask-extension
# Move "Enable Opensea API" and "NFT Autodetect" into Security & Privacy settings

## Explanation
Moves existing Opensea API and NFT Autodetect settings from the Experimental tab to the Security & Privacy tab
Fixes https://github.com/MetaMask/metamask-extension/issues/20062
Progress: https://github.com/MetaMask/metamask-extension/issues/20062#issuecomment-1654196908



## Screenshots/Screencaps



### Before
<img width="300" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/d8cb9972-576a-4014-bf22-0476acc5fe00">
<img width="300" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/95cbf499-258c-40a5-b889-5c788d469674">




### After


<img width="300" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/55095326-7348-4195-bfb1-a207447089d1">
<img width="300" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/ac2c5df5-c2fa-4028-b2aa-02257fd2cd6d">

## Manual Testing Steps


Open extension; go to Settings
Verify that the settings ARE NOT in Experimental tab
Verify that the settings ARE in Security & Privacy tab
Verify that searching with settings search bar will link directly to the setting

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
